### ISSUE TITLE
S3Trigger access denied

### ISSUE THREAD

**Issue by duncangroenewald** [aws-samples/amplify-photo-gallery-workshop#49]
I tried following the tutorial but the S3Trigger fails with an access denied when calling S3.getObject() - this is from the original version in the online tutorial.

Is it necessary to separately give the S3Trigger permission to access the S3 Bucket ?  If so where is this step performed.

I tried simply using 'amplify function update' and adding permission for access to 'storage' but then get an error in the amplify-cli.

**Comment by rydes01** [comment]
I had the same problem. In my case it was because I deviated from the tutorial by skipping the Amazon Rekognition integration. In doing so I missed the important commands to be executed at the bottom of the "Processing photos with the S3 Trigger Lambda function" page. I went back and executed the relevant commands --- which are to "ensure that the generated IAM policies don’t have a naming conflict" --- and the error went away.
